osu.Desktop.Tests/Visual/TestCaseHitObjects.cs

```diff
@@ -110,10 +109,7 @@ private void add(DrawableOsuHitObject h)
             h.Depth = depth++;
 
             if (auto)
-            {
                 h.State = ArmedState.Hit;
-                h.Judgement = new OsuJudgement { Result = HitResult.Hit };

```

Comment:
Pretty sure I uniformly removed this stuff from all test cases because idk how to handle this.